components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIImportUtil.java

```diff
@@ -177,14 +185,14 @@ private static String getLifeCycleAction(String tenantDomain, String currentStat
      */
     private static String loadSwaggerFile(String pathToArchive) throws IOException {
 
-        if (CommonUtil.checkFileExistence(pathToArchive + APIImportExportConstants.YAML_SWAGGER_DEFINITION_LOCATION)) {
+        if (checkFileExistence(pathToArchive + APIImportExportConstants.YAML_SWAGGER_DEFINITION_LOCATION)) {
```

Comment:
Since CommonUtil is a public util class and we are calling static methods, better to follow the convention `CommonUtil.checkFileExistence`

Reply:
resolved